nekoyume/Assets/_Scripts/UI/Module/IAP/IAPShopProductCellView.cs

```diff
@@ -258,6 +259,26 @@ public bool IsBuyable()
             return true;
         }
 
+        public bool IsNotification()
+        {
+            if (!_data.IsFree)
+                return false;
+
+            if (!IsBuyable())
+                return false;
+
+            if (dimObj.activeSelf)
+                return false;
+
+            if (_data.RequiredLevel == null)
+                return true;
+
+            if (_data.RequiredLevel.Value < States.Instance.CurrentAvatarState.level)
+                return true;
```

Comment:
```suggestion
            if (!_data.IsFree)
            {
                return false;
            }

            if (!IsBuyable())
            {
                return false;
            }

            if (dimObj.activeSelf)
            {
                return false;
            }

            if (_data.RequiredLevel == null)
            {
                return true;
            }

            if (_data.RequiredLevel.Value < States.Instance.CurrentAvatarState.level)
            {
                return true;
            }
```
if가 많아서 스코프를 붙이면 구분이 쉬울 것 같습니다.